project-base/storefront/components/Blocks/Adverts/Adverts.tsx

```diff
@@ -1,10 +1,7 @@
-import { ExtendedNextLink } from 'components/Basic/ExtendedNextLink/ExtendedNextLink';
-import { Image } from 'components/Basic/Image/Image';
+import { AdvertImage } from './AdvertImage';
 import { Webline } from 'components/Layout/Webline/Webline';
-import { TypeAdvertsFragment } from 'graphql/requests/adverts/fragments/AdvertsFragment.generated';
 import { useAdvertsQuery } from 'graphql/requests/adverts/queries/AdvertsQuery.generated';
 import { TypeCategoryDetailFragment } from 'graphql/requests/categories/fragments/CategoryDetailFragment.generated';
-import { Fragment } from 'react';
 import { twJoin } from 'tailwind-merge';
 
 type PositionNameType =
```

Comment:
This is defined, but not used in many places (where it could be beneficial). For example where the queries are called. Would you say it would be helpful to use it there, or not really?

Reply:
I think it would be beneficial if I needed to wrap two elements and pass a key inside the `map` function. I don't see the case here, thus not needed. Or am I missing something?